src/OrchardCore.Themes/TheAdmin/Views/NavbarLink.cshtml

```diff
@@ -0,0 +1,5 @@
+<li class="nav-item">
+    <a href="@Url.Content("~/")" class="nav-link" data-bs-toggle="tooltip" data-bs-placement="bottom" title="@T["Visit Site"]" role="button">
+        <i class="fa-solid fa-fw fa-external-link" aria-hidden="true"></i>
+    </a>
+</li>
```

Comment:
```suggestion
</li>

```

Reply:
Done in 7ef8072332bae0144fde567e9e229eac1b3ac793